### PR TITLE
Categorical sliders

### PR DESCRIPTION
**Pull request by drg101**
I also folded in a change to make the defensive stuff come up on the right, because it could be obscured by the menu on the left.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "build": "webpack --mode 'production'",
     "buildDev": "webpack --mode 'development'",
     "watch": "webpack --mode 'development' --watch",
-    "serve": "npm run watch & live-server --ignore='**/node_modules/**,**/src/**,**/index.js/**'",
+    "serve": "npm run watch & live-server --ignore='**/node_modules/**,**/src/**,index.js'",
```
Comment (drg101) [inline package.json:10]: Also, the only file which still can have the double reload issue is `index.js` so I'm trying this change out.

**Comment by pierce-smith1** [comment]
One UI comment: is it possible to place the current range string (e.g. "Relatively Moderate - Very High") on its own line? The interface starts looking kind of awkward when this happens:
image
I don't have any major comments on the format of the slider specification from a charting side, it should work fine to piggyback off of what you put in for the categorical plots.

**Comment by drg101** [comment]
@pierce-smith1 Addressed the issue, this actually made a lot of the UI look nicer.

**Review by pierce-smith1 — APPROVED** [review]
Nice

**Pull request merged**